src/procstat.c

```diff
@@ -731,25 +732,46 @@ int procstat_create_simple(struct procstat_context *context,
 	return -1;
 }
 
-void procstat_u64_series_add_point(struct procstat_series_u64 *series, uint64_t value)
+bool is_reset(struct procstat_series_u64 *series)
```

Comment:
@alexs-lb change the name to is_reset_series like you used specific name for histogram.

Reply:
done